frontend/packages/data-portal/app/components/Dataset/RunsTable.tsx

```diff
@@ -61,6 +63,25 @@ export function RunsTable() {
     [searchParams, setSingleDatasetHistory],
   )
 
+  const getRunUrl = useCallback(
+    (id: number) => {
+      const url = createUrl(`/runs/${id}`)
```

Comment:
nit - since we're only using the path and the query params, do we need `createUrl()`? what if `carryOverFilterParams()` just returned a new `URLSearchParams` and we did ```return `/runs/${id}\` + urlSearchParams.toString()```?